File: sendsms.java
URL url = new URL("https://gateway.standingtech.com/api/v3/sms/send");
HttpURLConnection http = (HttpURLConnection)url.openConnection();
http.setRequestMethod("POST");
http.setDoOutput(true);
http.setRequestProperty("Authorization", "Bearer API_KEY");
http.setRequestProperty("Content-Type", "application/json");
http.setRequestProperty("Accept", "application/json");

String data = "{\"recipient\":\"9647xxxxxxxxx\",\"sender_id\":\"SenderID\",\"type\":\"plain\",\"message\":\"This is a test message\"\n}";

byte[] out = data.getBytes(StandardCharsets.UTF_8);

OutputStream stream = http.getOutputStream();
stream.write(out);

System.out.println(http.getResponseCode() + " " + http.getResponseMessage());
http.disconnect();

File: sendsms.cs
using System;
using System.Net.Http;
using System.Text;
class Program
{
    static void Main()
    {
        var client = new HttpClient();
        client.DefaultRequestHeaders.Add("Authorization", "Bearer API_KEY");
var data = @"{
""recipient"":""9647xxxxxxxxx"",
""sender_id"":""SENDERID"",
""type"":""plain"",
""message"":""This is test using C# to send SMS""
}";

var content = new StringContent(data, Encoding.UTF8, "application/json");

var response = client.PostAsync("https://gateway.standingtech.com/api/v4/sms/send", content).Result;
var responseContent = response.Content.ReadAsStringAsync().Result;
Console.WriteLine(responseContent);
    }
}
